stock_product_qty_by_packaging/models/stock_move.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# @author: Sébastien Alix <sebastien.alix@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
```

Comment:
the license is mixed, probably my fault: could you replace "LGPL" everywhere w/ "AGPL"?